### PR TITLE
Simplify setup guide due to single config file

### PR DESCRIPTION
**Pull request by timmit99**
Simplification now that there's no display button version.

### CODE REVIEW

**Comment by timmit99** [comment]
@KiloQubit can you approve this?

File: V0_Display/Documentation/Setup_and_Flashing_Guide.md [hunk 1]
```diff
@@ -49,5 +49,5 @@
 
 11) After completion `ls /dev/serial/by-id/*` should return a device begining with `/dev/serial/by-id/usb-Klipper_stm32f042x6...`
 
-Your board should now be usable with Klipper. Use the example config files to get started
-Best optioon is to copty the config you want to use into the same directory as `printer.cfg` then add `[include NAME_OF_CONFIG.CFG]` to the end of your `printer.cfg` to include the file.
+Your board should now be usable with Klipper. Use the example config file to get started
+Best optioon is to copy the config file into the same directory as `printer.cfg` then add `[include DisplayEncoder.cfg]` to the end of your `printer.cfg` to include the file.
```
Comment (KiloQubit) [inline V0_Display/Documentation/Setup_and_Flashing_Guide.md:53]: s/optioon/option/

**Review by KiloQubit — CHANGES_REQUESTED** [review]

**Review by KiloQubit — APPROVED** [review]

**Comment by timmit99** [comment]
> @KiloQubit can you approve this?

**Comment by KiloQubit** [comment]
> @KiloQubit can you approve this?

I already did last night.  I have no powers here, no write access to this repo.

**Comment by timmit99** [comment]
@john-- HALP

**Review by raymondh2 — APPROVED** [review]

**Pull request merged**